lib/database_connection.rb

```diff
@@ -0,0 +1,12 @@
+require 'pg'
+
+class DatabaseConnection
+  def self.setup(dbname)
+    @connection = PG.connect(dbname: dbname)
+  end
+
+  def self.query(sql)
+    @connection.exec(sql)
```

Comment:
I get an error message when I try to post something containing this character: '
" PG::SyntaxError at /
ERROR: syntax error at or near "t" LINE 1: ...ERT INTO peeps VALUES (DEFAULT, DEFAULT, 'why can't post pee... ^
file: database_connection.rb location: async_exec line: 9 "

I coldn't find the problem yet

Reply:
Good catch. I did not take any measures to sanitise user input, which I should have done. Adding it to the list of possible improvements!

Reply:
What you have demonstrated is an example of SQL injection, similar to this: 
https://www.netsparker.com/blog/web-security/sql-injection-cheat-sheet/#ByPassingLoginScreens